production.ini

```diff
@@ -223,6 +223,9 @@ use = egg:bodhi-server
 # A list of non-person users
 # system_users = bodhi autoqa taskotron
 
+# The max number of mash threads running at the same time
+# max_concurrent_mashes = 2
```

Comment:
I think there's a "Mash settings" section (or something similarly named) where the pungi settings are - can we move this there? It's not a formal ini section, but there's a comment block to group them together.